aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -83,23 +84,109 @@ void pow_scalar_tensor_impl(TensorIteratorBase& iter, c10::complex<value_t> base
   });
 }
 
+/* complex<Half> support impl */
+const char pow_scalar_base_name[] = "pow_scalar_base_kernel";
+template <>
+void pow_scalar_tensor_impl(TensorIteratorBase& iter, c10::complex<at::Half> base) {
+  using scalar_t = c10::complex<at::Half>;
+  using opmath_t = at::opmath_type<scalar_t>;
+  // For complex, thrust::pow uses the identity
+  // pow(a, b) = exp(log(a) * b)
+  const auto fct = std::log(opmath_t{base});
+#if AT_USE_JITERATOR()
+  static const auto pow_kernel_string =
+      jiterator_stringify(template <typename T> T pow_scalar_base_kernel(T exp, T fct) {
+        return std::exp(fct * exp);
+      });
+  jitted_gpu_kernel<pow_scalar_base_name, scalar_t, scalar_t, 1>(
+      iter,
+      pow_kernel_string,
+      /*scalar_pos=*/at::cuda::jit::BinaryFuncVariant::NoScalar,
+      /*scalar_val=*/0,
+      /*extra_args=*/std::make_tuple(fct));
+#else
+  gpu_kernel(iter, [=] GPU_LAMBDA(scalar_t exp) -> scalar_t {
+    return std::exp(fct * opmath_t{exp});
+  });
+#endif
+}
+
+namespace {
+
+/* complex<Half> support impl */
+const char pow_name[] = "pow_kernel";
```

Comment:
Did try it without kernel here and in jiterator string, it was failing to compile with NVRTC (didn't investigate into that much though)

Reply:
np, can leave it as is